packages/grafana-ui/src/components/Collapse/Collapse.tsx

```diff
@@ -71,7 +72,14 @@ const getStyles = (theme: GrafanaTheme2) => ({
     label: 'collapse__header',
     padding: theme.spacing(1, 2, 1, 2),
     display: 'flex',
-    transition: 'all 0.1s linear',
+    ...handleReducedMotion(
+      {
+        transition: 'all 0.1s linear',
+      },
```

Comment:
I don't think this transition here even does anything. 

Looking at the code I believe the only thing it transitions is the `display` property, which can't animate anyway.

I think we just get rid of this.

https://github.com/grafana/grafana/blob/7dc6540ead3a03b77308f079b184c5fb5f01020d/packages/grafana-ui/src/components/Collapse/Collapse.tsx/#L70-L79